omod/src/main/java/org/openmrs/module/bahmni/ie/apps/service/impl/BahmniFormServiceImpl.java

```diff
@@ -136,6 +145,35 @@ private void updateFormResourceWithLatestVersion(Form form){
         return bahmniFormList;
     }
 
+    @Override
+    public ExportResponse getFormsByListOfUuids(List<String> formUuids) {
+        List<Form> formList = bahmniFormDao.getAllFormsByListOfUuids(formUuids);
+        List<BahmniFormData> bahmniFormDataList = new ArrayList<>();
+        List<String> errorFormNames = new ArrayList<>();
+        for (Form form : formList) {
+            try {
+                bahmniFormDataList.add(getBahmniFormData(form));
+            }
+            catch (Exception e){
+                errorFormNames.add(form.getName()+"_"+form.getVersion());
```

Comment:
Check styles are missing(spaces here). Please fix in other places also if required.